ui/raidboss/data/06-ew/trial/hydaelyn-ex.ts

```diff
@@ -20,23 +20,27 @@ const storedMechanicsOutputStrings = {
   spread: Outputs.spread,
   groups: {
     en: 'Healer Groups',
+    ko: '힐러 그룹 쉐어',
   },
   stack: {
     en: 'Party Stack',
+    ko: '쉐어',
   },
 };
 
 const crystallizeOutputStrings = {
   ...storedMechanicsOutputStrings,
   crystallize: {
     en: 'Crystallize: ${name}',
+    ko: 'Crystallize: ${name}', // FIXME
```

Comment:
```suggestion
```

If you can't translate it, I think it's better to leave it.

Reply:
you're right. I fixed it while rebasing, please check it.

Reply:
Thanks! Sorry for the small correction, but I just wanted more to let you know for the future.  :)